cmd/kops/update.go

```diff
@@ -32,7 +32,7 @@ var (
 
 	updateExample = templates.Examples(i18n.T(`
 		# After cluster has been created, configure it with:
-		kops update cluster k8s.cluster.site --yes --state=s3://kops-state-1234
+		kops update cluster k8s-aws-cluster.cluster.site --yes --state=s3://my-state-store
```

Comment:
might as well make this consistent with every other command's examples
```suggestion
		kops update cluster k8s-cluster.example.com --yes --state=s3://my-state-store
```

Reply:
Thought I did :)

Reply:
Done.